### DISCUSSION TITLE
Android Filename Date/Time

### DISCUSSION THREAD

**Discussion by j0enathan** [dimonovdd/Xamarin.MediaGallery#54] [Q&A] [answered]
It appears for Android a timestamp is concatenated on to the filename. Any way to save a file with the exact filename that is provide without any additional text?

**Comment by dimonovdd** [comment, marked as answer]
@j0enathan Hi
Oh no, it described in Readme file. We get exception if trying to save files with the same name. We also not request a permission to storage reading, so we don’t know with what names files were already saved.

**Comment by viciousnemesis** [comment]
It seems like it would be nice to request the needed permissions before you read to use that functionality, remove the timestamps, and simply rewrite any existing file.
